test/bigtest/tests/units/settings/JobProfilesForm/JobProfilesForm-test.js

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+import { noop } from 'lodash';
+import sinon from 'sinon';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Paneset } from '@folio/stripes/components';
+import { mountWithContext } from '@folio/stripes-data-transfer-components/interactors';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+import stripesComponentsTranslations from '@folio/stripes-components/translations/stripes-components/en';
+
+import translations from '../../../../../../translations/ui-data-export/en';
+import { JobProfilesForm } from '../../../../../../src/settings/JobProfiles/JobProfilesForm';
+import { JobProfilesFormInteractor } from './interactor';
+
+describe('JobProfilesForm', () => {
+  const translationsProperties = [
+    {
+      prefix: 'ui-data-export',
+      translations,
+    },
+    {
+      prefix: 'stripes-components',
+      translations: stripesComponentsTranslations,
+    },
+  ];
+
+  const mappingProfiles = [
+    {
+      value: 'id1',
+      label: 'name1',
+    },
+    {
+      value: 'id2',
+      label: 'name2',
+    },
+  ];
+
+  const form = new JobProfilesFormInteractor();
+
+  describe('rendering job profiles form with stubbed handlers and loaded mapping profiles', function () {
+    const handleSubmitSpy = sinon.spy();
+    const handleCancelSpy = sinon.spy();
+
+    beforeEach(async function () {
+      handleSubmitSpy.resetHistory();
+      handleCancelSpy.resetHistory();
+
+      await mountWithContext(
+        <Paneset>
+          <Router>
+            <JobProfilesForm
+              onSubmit={handleSubmitSpy}
+              onCancel={handleCancelSpy}
```

Comment:
Please move callbacks to the bottom. Same [here](https://github.com/folio-org/ui-data-export/pull/71/files#diff-46a900cdd87b6cf96bbc385ee7eb1894R185), [here](https://github.com/folio-org/ui-data-export/pull/71/files#diff-915f9e1b92d4f1f79a522a214a56ace0R31).